ddtrace/contrib/logging/patch.py

```diff
@@ -1,31 +1,74 @@
 import logging
 
-from ddtrace import config
+import ddtrace
 
-from ...helpers import get_correlation_ids
+from ...constants import ENV_KEY, VERSION_KEY
 from ...utils.wrappers import unwrap as _u
 from ...vendor.wrapt import wrap_function_wrapper as _w
 
-RECORD_ATTR_TRACE_ID = 'dd.trace_id'
-RECORD_ATTR_SPAN_ID = 'dd.span_id'
-RECORD_ATTR_VALUE_NULL = 0
+RECORD_ATTR_TRACE_ID = "dd.trace_id"
+RECORD_ATTR_SPAN_ID = "dd.span_id"
+RECORD_ATTR_ENV = "dd.env"
+RECORD_ATTR_VERSION = "dd.version"
+RECORD_ATTR_SERVICE = "dd.service"
+RECORD_ATTR_VALUE_ZERO = 0
+RECORD_ATTR_VALUE_EMPTY = ""
 
-config._add('logging', dict(
-    tracer=None,  # by default, override here for custom tracer
-))
+ddtrace.config._add("logging", dict(tracer=None,))  # by default, override here for custom tracer
+
+
+def _get_current_span(tracer=None):
+    """Helper to get the currently active span"""
+    if not tracer:
+        tracer = ddtrace.tracer
+
+    if not tracer.enabled:
+        return None
+
+    return tracer.current_span()
 
 
 def _w_makeRecord(func, instance, args, kwargs):
+    # Get the LogRecord instance for this log
     record = func(*args, **kwargs)
 
-    # add correlation identifiers to LogRecord
-    trace_id, span_id = get_correlation_ids(tracer=config.logging.tracer)
-    if trace_id and span_id:
-        setattr(record, RECORD_ATTR_TRACE_ID, trace_id)
-        setattr(record, RECORD_ATTR_SPAN_ID, span_id)
+    # Get the currently active span, if there is one
+    span = _get_current_span(tracer=ddtrace.config.logging.tracer)
+
+    # Inject `dd.version`
+    # Order of precedence:
+    #   - `version` tag on the currentl active span
```

Comment:
nit: small typo in `currently`; same below as well

Reply:
```suggestion
    #   - `version` tag on the currently active span
```